aas_editor/dialogs.py

```diff
@@ -105,7 +105,7 @@ def withDetailedText(cls, parent, text: str):
 class AddDialog(QDialog):
     """Base abstract class for custom dialogs for adding data"""
     REC = QGuiApplication.primaryScreen().geometry()
-    MAX_HEIGHT = int(REC.height() * 0.9)
+    MAX_HEIGHT = int(REC.height() * 0.6)
```

Comment:
The max possible height of the dialog should stay as it was. 

The problem should be solved as follows: When a new dialog is created and the dialog is placed with default setting or with saved INITIAL_POSITION (See line 143), we should calculate what the max height of a dialog shoud be to not cross monitor boundaries. We compare then the calculated height value with the current one and adjust it if needed.

May be there are already convinience funcs in Qt to do that. Please check